ui/lib/core/addon/components/info-table-row.hbs

```diff
@@ -6,7 +6,7 @@
 {{#if (or (has-block) this.isVisible)}}
   <div class="info-table-row" data-test-component="info-table-row" ...attributes>
     <div
-      class="column is-one-quarter {{if this.hasLabelOverflow 'label-overflow'}}"
+      class="column {{or @labelWidth 'is-one-quarter'}} {{if this.hasLabelOverflow 'label-overflow'}}"
```

Comment:
I'm probably reading this wrong, but to me or is (this _or_ that) for a conditional. Are we missing the "that" to show the is-one-quarter?

Reply:
if a `labelWidth` is passed in, it will override the default of `is-one-quarter`

Reply:
Oh... TIL. Interesting syntax on handlebars part but I like it!